packages/core/src/components/select/select.scss

```diff
@@ -53,26 +75,60 @@
         color: helpers.color('content-disabled');
       }
     }
+  }
 
-    &.-borderless {
-      @include mixins.with-inner-focus('action');
-      @include with-padding(); // no border width because border will be unset
+  .ods-select-custom.-open + .ods-icon {
+    transform: rotate(180deg);
+  }
 
-      background-color: transparent;
-      border: 0;
+  .ods-select-custom-dropdown {
+    @include mixins.with-inner-focus('action');
 
-      &:hover {
-        background-color: helpers.color('background-action-subtle-hover');
-      }
+    background-color: helpers.color('background-input');
+    border-radius: helpers.border-radius('small');
+    box-sizing: border-box;
+    display: none;
+    margin-top: helpers.space(0.5);
+    padding: 4px; // spread set on `with-inner-focus` mixin
+    position: absolute;
+    width: 100%;
 
-      &:disabled {
-        background-color: helpers.color('background-disabled');
-      }
+    > * {
+      box-sizing: inherit;
     }
 
-    &.-borderless.-invalid,
-    &.-borderless.-touched:invalid {
-      @include mixins.with-inner-focus('negative');
+    &.-open {
+      display: block;
+    }
+  }
+
+  .ods-select-custom-option {
+    @include helpers.font('300-regular');
+    @include mixins.interactive();
+
+    align-items: center;
+    appearance: none;
+    background-color: transparent;
+    border: 0;
+    color: helpers.color('content-main');
+    display: flex;
+    outline: none;
+    padding: helpers.space(1.5) helpers.space(2);
+    text-align: left;
+    width: 100%;
+
+    &.-focused {
```

Comment:
Can this not be `:focus`?

Reply:
It cannot, because during keyboard navigation a class is moved from item to item.

That is, a state keeps which option should be focused on, then option itself adds `.-focused` if it's the one that should be focused on.

Reply:
Interesting. I don't think I get the full picture but I'll continue to explore the component later.